### PR TITLE
[ARCTIC-504]When the catalog name contains line-through, terminal executes sql with an error

### LINKED ISSUE #504: [Bug]: When the catalog name contains line-through, terminal executes sql with an error
### What happened?

image
When the catalog name contains line-through, terminal executes sql with an error

### Affects Versions

Versions earlier than 0.3.2

### What engines are you seeing the problem on?

AMS

### How to reproduce

Terminal executes sql, when the catalog name contains line-through

### Relevant log output

_No response_

### Anything else

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by shendanfengg**
Resolve #504 

## Why are the changes needed?

* Resolve catalog names with special characters may lead terminal execute sql failed

## Brief change log
  - *Add back-quotes when execute "use catalog" 

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (no)

### CODE REVIEW

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ public static SesssionInfo executeSql(String catalog, String sqls) {
         }
         try {
           sqlRunningInfo.getLogs().add(df.format(new Date()) + " use " + catalog);
-          spark.sql("use " + catalog);
+          spark.sql("use " + "`" + catalog + "`");
```
Comment (YesOrNo828) [inline ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java:152]: PLS uses the `static final` keywords to define the String parameter: "`".

e.g. private static final String QUOTE = "`";

**Review by YesOrNo828 — COMMENTED** [review]
@shendanfengg, is there any other places should also quote the catalog?

File: ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java (hunk 1 above)
Comment (shendanfengg) [reply ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java]: > @shendanfengg, is there any other places should also quote the catalog?

No need to quote other place,only ams-server terminal needs this change

Comment (shendanfengg) [reply ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/TerminalService.java]: > PLS uses the `static final` keywords to define the String parameter: "`".
> 
> e.g. private static final String QUOTE = "`";

It has been fixed

**Review by YesOrNo828 — APPROVED** [review]
LGTM

**Pull request merged**